PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Infrastructure/Resources/SqlQueries.resx

```diff
@@ -0,0 +1,211 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
```

Comment:
Rework as in Artem or Nazar repository so that u use static class with sql resources